stackslib/src/net/codec.rs

```diff
@@ -276,7 +277,17 @@ impl BlocksInvData {
     }
 
     pub fn compress_bools(bits: &Vec<bool>) -> Vec<u8> {
```

Comment:
In general it's preferable to use slices rather than `&Vec` for function parameters

```suggestion
    pub fn compress_bools(bits: &[bool]) -> Vec<u8> {
```

Also, this function looks like it's reimplementing `BitVec::try_from::<bool>()`

Reply:
This is part of a legacy code path that's live in production, which I'm _very_ keen not to touch